### DISCUSSION TITLE
Medium link not working!

### DISCUSSION THREAD

**Discussion by Deepubhatt** [academicpages/academicpages.github.io#2440] [Q&A] [answered]
So I recently forked and made my portfolio site using your theme, I added my socials like LinkedIn, GitHub, email, etc. But when I gave my Medium username to `_config.yml`, on clicking the link on the website, it showed that no page was found.

Is there any workaround for this? As I didn't face any issues with my other socials.

Thank you for your response!

**Comment by rjzupkoii** [comment]
There might be a bug in the template if Medium changed things up, what's the link to your Medium page?

**Reply by Deepubhatt** [reply]
Thanks for the reply! Here is the link to my profile: https://medium.com/@deepakbhatt17

I tried to give the username using both `@deepakbhatt17` and `deepakbhatt17`, but they did not work. Appreciate your time.

**Reply by rjzupkoii** [reply, marked as answer]
Alright, I see what is going on. The `_config.yml` notes that the Medium social wants a URL and the code in `author-profile.html` wants that as well. So if you put the full URL in for that it should work fine - but if medium has URLs with a fixed username then we can just update that to take  username (like most of the other sites) instead.

**Reply by Deepubhatt** [reply]
Thanks, man! It worked. It was such a simple solution, sorry to bother you.

Actually, I thought of putting the link also, but then I thought it wouldn't work as in the comments it asked for a username. Again, thanks for the help.
